lib/List.php

```diff
@@ -26,7 +26,7 @@ class Turba_List implements Countable
     /**
      * The field to compare objects by.
      *
-     * @var string
+     * @var iterable
```

Comment:
Looking at code using this variable, it cannot be a string but an array or a list class.